.github/workflows/lint.workflow.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  push:
+    branches-ignore:    
+      - master
+      - gh-pages
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Set up Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18.x
+
+    - name: Install dependencies
+      run: npm ci 
+
+    - name: Run lint
+      run: npm run lint
+      
```

Comment:
Whoops; this has an extra space so the newline isn't applying.

Reply:
Thank you for catching that, I've gone ahead and remove the extra spaces at the end of that.